src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -24,15 +24,28 @@
     = render partial: 'webui2/shared/buildresult_box', locals: { project: @project.name }
   .list
     .card
-      %h5.card-header
-        Packages
-        = "(#{@packages.length})"
-      = render partial: 'project_packages'
+      .bg-light
+        %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap#packages-tabs{ role: 'tablist' }
+          %li.nav-item
+            %a.nav-link.active#packages-tab{ href: '#packages', role: 'tab', data: { toggle: 'tab' },
+            aria: { controls: 'packages', selected: 'true' } }
+              Packages (#{@packages.length})
+          - if @ipackages.present?
+            %li.nav-item
+              %a.nav-link#inherited-packages-tab{ href: '#inherited-packages', role: 'tab', data: { toggle: 'tab' },
+              aria: { controls: 'inherited-packages', selected: 'false' } }
+                Inherited Packages (#{@ipackages.length})
+      .tab-content#packages-tabs-content
+        .tab-pane.fade.show.active#packages{ role: 'tabpanel', aria: { labelledby: 'packages-tab' } }
+          = render partial: 'project_packages', locals: { project: @project, packages: @packages }
+        .tab-pane.fade#inherited-packages{ role: 'tabpanel', aria: { labelledby: 'inherited-packages-tab' } }
+          = render partial: 'project_inherited_packages', locals: { inherited_packages: @ipackages }
   .comments
     .card
       %h5.card-header
-        Comments for
-        = @project
-        (#{ @comments.length})
+        Comments for #{@project} (#{@comments.length})
       .card-body#comments
         = render partial: 'webui2/webui/comment/show', locals: { commentable: @project }
+
+= content_for :ready_function do
+  $('#packages-table,#inherited-packages-table').DataTable();
```

Comment:
Take a look at https://github.com/openSUSE/obs-patterns/pull/42 It solves a bug related to the search field

Reply:
@ana06 fixed it